core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/file/UserFileResource.scala

```diff
@@ -248,39 +220,16 @@ class UserFileResource {
 
   /**
     * This method deletes a file from a user's repository
-    * @param fileName the name of file being deleted
-    * @param ownerName the name of the file's owner
+    * @param fid id of the file
     * @return
     */
   @DELETE
-  @Path("/delete/{fileName}/{ownerName}")
-  @RolesAllowed(Array("REGULAR", "ADMIN"))
+  @Path("/delete/{fid}")
```

Comment:
For the delete opertion, does the caller not need to have access to the file?

Reply:
Added access permission checks.